src/clientiface.cpp

```diff
@@ -60,6 +60,24 @@ void RemoteClient::ResendBlockIfOnWire(v3s16 p)
 	}
 }
 
+LuaEntitySAO *getAttachedObject(PlayerSAO *sao, ServerEnvironment *env) {
+	LuaEntitySAO *lsao = nullptr;
+	if (sao->isAttached()) {
```

Comment:
can you invert condition & do a return, this permits to reduce indentation level as it's the global fct condition ?

Reply:
Done